core/macros/macros-api/tsconfig.json

```
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
+    "rootDir": "src",
+    "outDir": "dist"
+  },
```

```suggestion
  "compilerOptions": {
    "rootDir": "src",
    "outDir": "dist",
    "emitDeclarationOnly": true,
    "noEmit": false,
    "declarationMap": true
  },
```

Currently, calling `tsc` does not emit code

---

The suggest is not applied anymore